src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (a) => {
+    return (b) => {
```

Comment:
Let's set better variables names.
```suggestion
  return (firstNumber) => {
    return (secondNumber) => {
```